src/Aquifer.Jobs/SendNewContentEmails.cs

```diff
@@ -0,0 +1,100 @@
+using Aquifer.Common;
+using Aquifer.Common.Clients;
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using Aquifer.Jobs.Configuration;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Options;
+using SendGrid.Helpers.Mail;
+
+namespace Aquifer.Jobs;
+
+public class SendNewContentEmails(AquiferDbContext dbContext, SendGridClient client, IOptions<ConfigurationOptions> options)
+{
+    [Function(nameof(SendNewContentEmails))]
+    public async Task Run([TimerTrigger("%NewContentEmail:CronSchedule%")] TimerInfo timerInfo, CancellationToken ct)
+    {
+        var today = DateTime.Today;
+        var firstOfThisMonth = new DateTime(today.Year, today.Month, 1);
+        var firstOfLastMonth = firstOfThisMonth.AddMonths(-1);
+
+        var subscribers = await dbContext.ContentSubscribers.Where(cs => cs.Enabled)
+            .Select(cs => new SubscriberInfo
+            {
+                Name = cs.Name,
+                Email = cs.Email,
+                UnsubscribeId = cs.UnsubscribeId,
+                Languages = cs.ContentSubscriberLanguages.Select(csl => csl.Language),
+                ParentResources = cs.ContentSubscriberParentResources.Select(cspr => cspr.ParentResource)
+            })
+            .ToListAsync(ct);
+
+        var alLanguageIds = subscribers.SelectMany(x => x.Languages.Select(l => l.Id));
+        var allParentResourceIds = subscribers.SelectMany(x => x.ParentResources.Select(pr => pr.Id));
+
+        var allNewItems = await dbContext.ResourceContentVersions.Where(x =>
+                x.IsPublished &&
+                x.Updated >= firstOfLastMonth &&
+                x.Updated <= firstOfThisMonth &&
+                alLanguageIds.Contains(x.ResourceContent.LanguageId) &&
+                allParentResourceIds.Contains(x.ResourceContent.Resource.ParentResourceId))
+            .Select(x => new UpdatedParentResources{ParentResourceId = x.ResourceContent.Resource.ParentResourceId,
+                DisplayName = x.ResourceContent.Resource.ParentResource.DisplayName,
+                LanguageId = x.ResourceContent.LanguageId,
+                EnglishDisplay = x.ResourceContent.Language.EnglishDisplay})
+            .ToListAsync(ct);
+
+        var htmlTemplate = dbContext.EmailTemplates.Single(t => t.Id == (int)EmailTemplate.AquiferMarketingNotification);
+
+        foreach (var subscriber in subscribers)
```

Comment:
I think we're missing some sort of logging on this. I think we should log to app insights an event for each email sent and what was sent.

Reply:
Pushed some changes for adding insights for WorkerServices. Going to add more detail for the event, but thought you could take a look at what I have right now while I put the details together